projects/addon-doc/components/main/main.template.html

```diff
@@ -9,6 +9,20 @@
     </div>
     <header tuiDocHeader>
         <ng-content select="tuiDocHeader" />
+
+        <button
+            appearance="outline"
+            aria-label="Switch directionality of the element's text"
+            iconEnd="@tui.a-large-small"
+            size="s"
+            tuiButton
+            type="button"
+            class="tui-doc-switch-direction"
+            (click)="changeTextDirection()"
+        >
+            <tui-icon icon="@tui.arrow-right-left" />
```

Comment:
Will it appear in all taiga family documentation ?
Could it be disabled ? For maskito documentation, for example